twitter_ads/campaign.py

```diff
@@ -278,6 +278,16 @@ def targeting_criteria(self, id=None, **kwargs):
         else:
             return TargetingCriteria.load(self.account, id, **kwargs)
 
+    def save(self):
+        # automatically_select_bid and bid_type are exclusive parameters
+        if self.automatically_select_bid and self.bid_type:
+            if self.bid_type == 'AUTO':
+                self.bid_type = None
+                self.automatically_select_bid = True
+            else:
+                self.automatically_select_bid = None
+        super(LineItem, self).save()
```

Comment:
Looks like this way of calling `super()` is python 2 specific. Will need to be refactored for python3. Also, is there a way to avoid having to call this given that its different from our existing paradigms?

Reply:
@tushdante From Python 3, we can _omit_ arguments for `super()` but it doesn't mean we should follow that rule. I like this way explicitly write what the arguments are to avoid confusion.

> is there a way to avoid having to call this given that its different from our existing paradigms?

I see your point but this is actually a better way from code quality point of view, I think. Otherwise, we keep writing duplicate code which we ideally should avoid.